{
	"id": "CVE-2019-19844",
	"vulnerabilities": [
		{
			"id": "CVE-2019-19844",
			"description": "Django before 1.11.27, 2.x before 2.2.9, and 3.x before 3.0.1 allows account takeover. A suitably crafted email address (that is equal to an existing user's email address after case transformation of Unicode characters) would allow an attacker to be sent a password reset token for the matched user account. (One mitigation in the new releases is to send password reset tokens only to the registered user email address.)",
			"severity": [
				{
					"type": "cvss_v31",
					"source": "security.alpinelinux.org",
					"cvss_v31": {
						"vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
						"base_score": 9.8,
						"base_severity": "CRITICAL",
						"temporal_score": 9.8,
						"temporal_severity": "CRITICAL",
						"environmental_score": 9.8,
						"environmental_severity": "CRITICAL"
					}
				}
			],
			"references": [
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.alpinelinux.org/vuln/CVE-2019-19844"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://docs.djangoproject.com/en/dev/releases/security/"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.gentoo.org/glsa/202004-17"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.netapp.com/advisory/ntap-20200110-0003/"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://www.debian.org/security/2020/dsa-4598"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://www.djangoproject.com/weblog/2019/dec/18/security-releases/"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "http://packetstormsecurity.com/files/155872/Django-Account-Hijack.html"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://groups.google.com/forum/#%21topic/django-announce/3oaB2rVH3a0"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/HCM2DPUI7TOZWN4A6JFQFUVQ2XGE7GUD/"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://seclists.org/bugtraq/2020/Jan/9"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://usn.ubuntu.com/4224-1/"
				}
			],
			"published": "2019-12-18T19:15:11Z",
			"modified": "2024-05-14T06:59:01.315103Z"
		}
	],
	"detection": [
		{
			"ecosystem": "alpine:3.10",
			"vulnerable": true,
			"package": {
				"name": "py-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.27-r0"
					}
				],
				"fixed": [
					"1.11.27-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.11",
			"vulnerable": true,
			"package": {
				"name": "py3-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.27-r0"
					}
				],
				"fixed": [
					"1.11.27-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.12",
			"vulnerable": true,
			"package": {
				"name": "py3-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.27-r0"
					}
				],
				"fixed": [
					"1.11.27-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.8",
			"vulnerable": true,
			"package": {
				"name": "py-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.27-r0"
					}
				],
				"fixed": [
					"1.11.27-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.9",
			"vulnerable": true,
			"package": {
				"name": "py-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.27-r0"
					}
				],
				"fixed": [
					"1.11.27-r0"
				]
			}
		}
	],
	"data_source": "alpine-osv"
}
